lib/jobi.js

```diff
@@ -14,10 +14,12 @@ const DEFAULT_OPTS = {
 // Shared Jobi settings symbols
 const sharedLogLevel  = Symbol.for('JOBI_sharedLogLevel');
 const sharedLogFormat = Symbol.for('JOBI_sharedLogFormat');
+const sharedLogPrefix = Symbol.for('JOBI_sharedLogPrefix');
```

Comment:
Think this should be localLogPrefix since it’s used per-instance rather than globally